articles/article-105-livelock.md

```diff
@@ -0,0 +1,19 @@
+<!-- BEGIN TITLE -->
+# Oops! My computer keeps freezing, could it be a livelock?
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Yes, It could be. A livelock is one of the reasons that might freeze your system. But, what is a livelock?  
+
+Suppose two gentlemen sitting in a restaurant and the waiter brings the order without enough cutlery. The first gentleman has a fork and the second one has a knife. To be able to eat, the first asks the second for the knife and at the same time the second asks the first for the fork. They exchange the cutlery, and the situation happens again; the first has the knife and the second has the fork. They keep exchanging the cutlery without eating. 
```

Comment:
And one more language thing: I think it should be: "Suppose two gentlemen *are* sitting ..."
And maybe: The first asks the second *one*

Reply:
You are correct, I added "are".
I am not sure about the second comment, as it is clear from the sentence and the sentence would be more complicated if we make it "The first one asks the second one ..."

Reply:
agreed. So for me this looks fine now. Thanks for all the changes.